{{ cookiecutter.project_name }}/pyproject.toml

```diff
@@ -18,12 +18,12 @@ keywords = []
 requires-python = ">=3.12"
 dependencies = []
 
+[project.optional-dependencies]
+test = ["pytest>=8.4.2", "pytest-cov>=6.3.0", "pytest-sugar>=1.1.1"]
+
```

Comment:
I need to do a bit of googling on this. I've automatically been using uv's `install --dev`, but I might need to change that.